manager/cve_handler.py

```diff
@@ -316,6 +316,21 @@ def _cve_details(cls, synopsis, args):
             retval['status_id'] = 0
             retval['status_text'] = None
 
+        # affected but not vulnerable systems count
+        # pylint: disable=singleton-comparison
+        not_vulnerable = (SystemVulnerabilities
+                          .select(fn.Count(SystemVulnerabilities.id))
+                          .join(CveMetadata, on=(SystemVulnerabilities.cve_id == CveMetadata.id))
+                          .join(SystemPlatform, on=(SystemVulnerabilities.system_id == SystemPlatform.id))
+                          .join(InsightsRule, on=(SystemVulnerabilities.rule_id == InsightsRule.id))
+                          .where((InsightsRule.active == True) & (InsightsRule.rule_only == True))
```

Comment:
After the PR check tests we found out there is one more condition missing.
The `when_mitigated` column from `system_vulnerabilities` table must be NULL. This is because this column is set when the vmaas detects there is vulnerable RPM not detected anymore. Affected but not vulnerable must have the vulnerable RPM installed.